test/arbitrary.test.ts

```diff
@@ -1,10 +1,12 @@
 import * as fc from '../src/index'
+import * as Strategies from '../src/strategies/index'
 import {it} from 'mocha'
 import {expect} from 'chai'
 
 describe('Arbitrary tests', () => {
   it('should return has many numbers has asked', () => {
     expect(fc.scenario()
+      .config(new Strategies.RandomCachedStrategy())
```

Comment:
Isn't there a default strategy?

Reply:
No. The default one is the BiasedRandomChachedStrategyWithShrinking.

Reply:
We should have tests specifically for each strategy.

Reply:
Yeah, we can do that. However, notice that the strategy can influence test performance. As such for every test case where shrinking is not needed we should use a _shrinkingless_ strategy to avoid additional computation no?

Reply:
Let the current tests unchaged. They should still work and serve as a good regression testing suite.

Create a new test suite for strategies.

Reply:
Instead of doing the proposed changes on this PR, I created an issue #144 to be picked later.